scripts/build-daemon.sh

```diff
@@ -0,0 +1,25 @@
+set -e
+set -o pipefail
+
+STAGE=$1
+
+if [ -z $STAGE ]; then
+    echo "This scripts expects the stage as a parameter";
+    exit 1;
+fi
+
+if [ -z "$ACCOUNT_ID" ]; then
+    echo "Please export a ACCOUNT_ID env var before running this";
+    exit 1;
+fi
+
+commit=`git rev-parse HEAD`;
+timestamp=`date +%s`;
+export DOCKER_IMAGE_TAG="$1-$commit-$timestamp";
```

Comment:
There is an important difference between what you're doing, and the way we did it in the past (in the `.github/workflows/deploy.yml` file that was deleted).

The difference is that the Git version tags won't be used as Docker tags anymore.

I believe we should keep the old behavior. Git tags should be used as Docker tags, because it's easier to identify the images later.

But regardless, we need to make sure the same patterns used to build the images here will be used by the rest of our automation tools to detect and deploy the images.

Reply:
I agree.

Added the `DOCKER_IMAGE_TAG` on the build.sh and using it on the codebuild script